src/module/Ui/templates/layout/en/serlo-home.phtml

```diff
@@ -99,7 +99,7 @@ echo $this->doctype();
                     <div class="home-row">
                         <p>
                             It is our vision to enable personalized learning and provide high
-                            quality educational resources worldwide &ndash; completely free of charge.
+                            quality educational resources<br>worldwide &ndash; completely free of charge.
```

Comment:
Does this work on any screen size? Manual line brakes should be avoided if possible.